<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SingleModeT</class>
 <widget class="QWidget" name="SingleModeT">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>650</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="Clock" name="clock">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>610</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="marksLabel">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>10</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="PauseWidget" name="pauseWidget" native="true">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>650</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="matrixButton">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>30</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>ShowMatrix</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hintButton">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>60</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>HintButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>701</x>
     <y>200</y>
     <width>91</width>
     <height>261</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <zorder>graphicsView</zorder>
  <zorder>marksLabel</zorder>
  <zorder>clock</zorder>
  <zorder>pauseWidget</zorder>
  <zorder>matrixButton</zorder>
  <zorder>hintButton</zorder>
  <zorder>statusLabel</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Clock</class>
   <extends>QLCDNumber</extends>
   <header location="global">clock.h</header>
  </customwidget>
  <customwidget>
   <class>PauseWidget</class>
   <extends>QWidget</extends>
   <header location="global">pausewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
